app/src/main/java/dev/bluehouse/enablevolte/MainActivity.kt

```diff
@@ -1,42 +1,37 @@
 package dev.bluehouse.enablevolte
 
 import android.content.Context
-import android.content.Intent
 import android.content.pm.PackageManager
 import android.content.res.Resources
-import android.hardware.radio.sim.Carrier
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.os.PersistableBundle
 import android.telephony.CarrierConfigManager
 import android.telephony.SubscriptionManager
 import android.telephony.SubscriptionManager.DEFAULT_SUBSCRIPTION_ID
-import android.telephony.SubscriptionManager.MAX_SUBSCRIPTION_ID_VALUE
 import android.telephony.TelephonyFrameworkInitializer
 import android.telephony.TelephonyManager
-import android.util.Log
-import android.widget.Toast
 import androidx.databinding.DataBindingUtil
 import com.android.internal.telephony.ICarrierConfigLoader
 import dev.bluehouse.enablevolte.databinding.ActivityMainBinding
 import org.lsposed.hiddenapibypass.HiddenApiBypass
 import rikka.shizuku.Shizuku
 import rikka.shizuku.ShizukuBinderWrapper
-import rikka.shizuku.ShizukuProvider
-import rikka.sui.Sui
 
 data class ShizukuData(
     var shizukuEnabled: Boolean,
     var shizukuGranted: Boolean,
-    var deviceIMSEnabled: Boolean,
-    var carrierIMSEnabled: Boolean,
+    var deviceVolteEnabled: Boolean,
+    var carrierVolteEnabled: Boolean,
```

Comment:
To keep consistency across method and variable names, let's change Volte to VoLTE.
```suggestion
    var deviceVoLTEEnabled: Boolean,
    var carrierVoLTEEnabled: Boolean,
```